static/usr/lib/systemd/system/remount-core-fs.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Reset propagation of initial mount points
+DefaultDependencies=no
+Before=local-fs-pre.target
+Before=local-fs.target
+Before=shutdown.target
+Wants=local-fs-pre.target
```

Comment:
`local-fs-pre.target` is pulled already by `local-fs.target`

Reply:
No. It only has `After=`. Typically the `-pre` targets are only added if needed.

Reply:
I see, thanks